recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "legacy-api-wrap" %}
-{% set version = "1.2" %}
+{% set version = "1.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Sadly, PyPI is not consistent with the `-` and `_`.

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/legacy_api_wrap-{{ version }}.tar.gz
```

---

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name | replace("-", "_") }}-{{ version }}.tar.gz
```